src/utils/event.ts

```diff
@@ -8,6 +8,7 @@ import {
 } from "date-fns";
 
 import { VenueEvent } from "types/venues";
+import { WithId, WithVenueId } from "./id";
```

Comment:
```suggestion

import { WithId, WithVenueId } from "./id";
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1538/commits/c87e9c8094ad8ceb7a56aaa668ecf94c895afba2